recipes/steinbock/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "steinbock" %}
+{% set version = "0.16.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/steinbock-{{ version }}.tar.gz
+  sha256: 0c1802a319255914fea00bf3c6bfde79f27ec536637e179d5f582525bfee56fe
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - steinbock = steinbock._cli:steinbock_cmd_group
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+    - setuptools-scm
+  run:
+    - anndata
+    - click
+    - click-log
+    - fcswrite
+    - h5py
+    - imageio
+    - networkx
+    - numpy
+    - opencv
+    - pandas
+    - python >=3.8
+    - scikit-image
+    - scipy
+    - tifffile
+    - xtiff
+
+test:
+  imports:
+    - steinbock
+  commands:
+    - pip list
+    # - steinbock
```

Let's have at least something to test that the entry point was created:

```suggestion
    - which steinbock
```